### PR TITLE
installation from helm - part 1

### PR DESCRIPTION
**Pull request by mitchdraft**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -165,36 +168,67 @@ endif
...
-	docker push $(DOCKER_REPO)/squash:$(VERSION)
-endif
+docker-push: docker
+	docker push $(CONTAINER_REPO_ORG)/plank-dlv:$(IMAGE_TAG) && \
+	docker push $(CONTAINER_REPO_ORG)/plank-gdb:$(IMAGE_TAG) && \
+	docker push $(CONTAINER_REPO_ORG)/squash:$(IMAGE_TAG)
+
+# TODO(mitchdraft) - replace this with a go lib when it's available
+$(OUTPUT_DIR)/buildtimevalues.yaml:
+	echo plank_image_tag: $(IMAGE_TAG) > $@
```
Comment (mitchdraft) [inline Makefile]: if we move this to a golib we can import these strings and reduce the risk of drift/bugs

File: Makefile [hunk 2]
```diff
@@ -165,36 +168,67 @@ endif
...
+	echo plank_image_repo: $(CONTAINER_REPO_ORG) >> $@
 
+.PHONY: mktest
```
Comment (mitchdraft) [inline Makefile]: oops, deleting this now

File: Makefile (hunk 1 above)
Comment (rickducott) [reply Makefile]: as discussed over slack, we should move away from this towards build id or commit / describe based

**Comment by mitchdraft** [comment]
now running with fresh plank containers!
```
Step #8 - "e2e-test": [1556216002] E2e Squash Suite - 2/2 specs Squash tests are running under the following conditions:
Step #8 - "e2e-test": plank repo: gcr.io/solo-public
Step #8 - "e2e-test": plank tag: d36be1f7-cef8-4db6-8007-e19f63a31eff
Step #8 - "e2e-test": values set from build tool
```
iteration will be safer/faster now

**Pull request merged**